express-get-json/index.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const gradesArray = [];
+for (const grade in grades) {
+  gradesArray.push(grades[grade]);
+}
```

Comment:
Try doing this in your route so that if the data model changes you don't have to manage this array too.